src/bp/diag/index.ts

```diff
@@ -164,6 +167,14 @@ const testConnectivity = async () => {
       process.env.BP_REDIS_SCOPE && printRow('Redis using scope', process.env.BP_REDIS_SCOPE)
       printRow('Botpress nodes listening on Redis', reply[1])
```

Comment:
Here we print the number of nodes listening for a given scope but above, we list all clients no matter their scope. 

I think we should either make the Redis diag report fully-scoped or non-scoped at all. What do you think would be the best?

Reply:
The number of subscribers is the number of nodes counted toward the licensing limit, so using the "global" number wouldn't be useful. Since we display the current scope on the previous line, I think it may be useful to display all "detected" scopes, since we might use that information to diagnose an issue (eg: an incorrectly scoped redis key)